src/views/ViewEditPatient.vue

```diff
@@ -3,50 +3,78 @@
     <NavBar />
 
     <div class="flex">
-      <SideBar :activeSection="activeSection" :id="id" :name="name" :age="age ? age : undefined"
-        @update:activeSection="setActiveSection" @openTryDeleteVisitModal="tryDeleteVisit = true" :isAdd="false" />
+      <SideBar
+        :activeSection="activeSection"
+        :id="id"
+        :name="name"
+        :age="age ? age : undefined"
+        @update:activeSection="setActiveSection"
+        @openTryDeleteVisitModal="tryDeleteVisit = true"
+        :isAdd="false"
+      />
       <div class="flex-grow">
-        <SubNavBar :id="id" :regDate="regDate" :queueNo="queueNo"
-          @openModal="openRecords" />
+        <SubNavBar
+          :id="id"
+          :regDate="patient?.admin.regDate"
```

Comment:
Why is this change needed? For ViewEditPatients i think patient.admin.regDate and patient.admin.queueNo are guaranteed to exist